src/command/pack.rs

```diff
@@ -9,7 +9,15 @@ pub fn pack(path: Option<String>, log: &Logger) -> result::Result<(), Error> {
     let crate_path = set_crate_path(path);
 
     info!(&log, "Packing up the npm package...");
-    npm::npm_pack(&crate_path)?;
+    match npm::npm_pack(&crate_path) {
+        Ok(r) => r,
+        Err(Error::Io { .. }) => {
+            return Err(Error::DirNotFound {
+                message: "Unable to find the pkg directory".to_owned(),
```

Comment:
Should this be more specific? Should it also print the path that the user entered? Not sure yet.